fujiji-client/src/components/NavBar/NavBar.jsx

```diff
@@ -1,115 +1,65 @@
+    <li className={styles.navItem}><a href="/#" className={styles.navLink}>Sign In</a></li>
+  </ul>
 );
 
 const UserGroup = (
-  <Menu>
-    <Button
-      display={{ base: 'none', md: 'inline-flex' }}
-      variant="solid"
-      colorScheme="teal"
-      size="sm"
-      mr={4}
-      leftIcon={<AddIcon />}
-      aria-label="post-listing-button"
-    >
-      Post
-    </Button>
-    <MenuButton
-      as={Button}
-      rounded="full"
-      variant="link"
-      cursor="pointer"
-      minW={0}
-      aria-label="profile-button"
-    >
-      <Avatar aria-label="profile-image" size="sm" alt="Profile image" />
-    </MenuButton>
-    <MenuList>
-      <MenuItem>Home</MenuItem>
-      <MenuItem
-        onClick={() => {
-          // TODO: route to My Listings page
-        }}
-      >
-        My Listings
-      </MenuItem>
-      <MenuDivider />
-      <MenuItem
-        onClick={() => {
-          // TODO: Implement Sign Out
-        }}
-      >
-        Sign Out
-      </MenuItem>
-    </MenuList>
-  </Menu>
+  <ul className={styles.navList}>
+    <li className={styles.navItem}><a href="/#" className={styles.navLink}>Home</a></li>
+    <li className={styles.navItem}><a href="/#" className={styles.navLink}>Search</a></li>
+    <li className={styles.navItem}><a href="/#" className={styles.navLink}>Post</a></li>
+    <li className={styles.navItem}><a href="/#" className={styles.navLink}>Favourites</a></li>
+    <li className={styles.navItem}><a href="/#" className={styles.navLink}>Sign Out</a></li>
+  </ul>
 );
 
 export default function NavBar({ isSignedIn = false }) {
   const rightNavGroup = isSignedIn ? UserGroup : SignInGroup;
+  const [toggleMenu, settoggleMenu] = useState(false);
+  const [showCloseIcon, setShowCloseIcon] = useState(false);
+  const menuClasses = toggleMenu
+    ? `${styles.showMenu} ${styles.navMenu}`
+    : styles.navMenu;
 
-  return (
-    <Box bg="gray.100" px={4}>
-      <Flex h={16} alignItems="center" justifyContent="space-between">
-        <Box fontWeight={600} fontSize="24px">
-          <Link
-            aria-label="Marketplace link"
-            px={2}
-            py={1}
-            href="/test/test"
-            _hover={{ textDecoration: 'none' }}
-          >
-            Fujiji
-          </Link>
-        </Box>
+  const closeIconClasses = showCloseIcon
+    ? `${styles.showCloseIcon} ${styles.navClose}`
+    : styles.navClose;
+
+  const openMenu = () => {
+    settoggleMenu(true);
+    setShowCloseIcon(true);
+  };
 
-        <Flex alignItems="center">
+  const closeMenu = () => {
+    settoggleMenu(false);
+    setShowCloseIcon(false);
+  };
+
+  return (
+    <div className={styles.header}>
+      <nav className={`${styles.grid} ${styles.nav}`}>
+        <div>
+          <a href="/#" className={styles.navLogo}>Fujiji</a>
+        </div>
+        <div className={styles.navToggle} id="navToggle" role="presentation" onClick={openMenu}>
+          <HamburgerIcon />
+        </div>
```

Comment:
Is it better if we use an Avatar/Empty profile image icon instead? I think a `Hamburger` menu does not tell the user if they signed in